app/status.go

```diff
@@ -172,11 +172,35 @@ func (a *App) SetStatusLastActivityAt(userId string, activityAt int64) {
 	a.SetStatusAwayIfNeeded(userId, false)
 }
 
+func (a *App) GetUserConnectionCount(userId string) int {
+	hub := a.GetHubForUserId(userId)
+	if hub != nil {
+		return hub.UserConnectionCount(userId)
+	}
+	return 0
+}
+
 func (a *App) SetStatusOnline(userId string, manual bool) {
 	if !*a.Config().ServiceSettings.EnableUserStatuses {
 		return
 	}
 
+	clusterCount := 0
```

Comment:
We can move that into an independent function, something like `GetUserCluserConnectionCount`